package.json

```diff
@@ -60,7 +60,7 @@
     "test:e2e:macos": "scripts/macos_e2e.sh 'test'"
   },
   "peerDependencies": {
-    "react": "^16.8",
+    "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1",
     "react-native": ">=0.59"
```

Comment:
@Krizzu: In other projects, we use:
```json
    "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1",
    "react-native": "^0.60.6 || ^0.61.5 || ^0.62.2 || ^0.63.2 || ^0.64.0 || 1000.0.0",
```
Should we also change `react-native` to be more specific? I'm not sure we can say that AsyncStorage will also work whenever TurboModules comes around.

Reply:
Yes, do won't relay on RN version, so please go ahead